### ISSUE TITLE
Dependency can not be resolved in SBT

### ISSUE THREAD

**Issue by achahal** [awslabs/aws-dynamodb-examples#6]
Sbt can not resolve latest dependencies

```
resolvers += Resolver.url("dynamodblocal", url("http://dynamodb-local.s3-website-us-west-2.amazonaws.com/release"))
libraryDependencies +=   "com.amazonaws" % "DynamoDBLocal" % "1.11.0.1" % "test"
```

---Errors---

> ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: com.amazonaws#DynamoDBLocal;1.11.0: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn]  Note: Unresolved dependencies path:
[warn]          com.amazonaws:DynamoDBLocal:1.11.0

**Comment by laughedelic** [comment]
You should use a [Maven-repository resolver](http://www.scala-sbt.org/0.13/docs/Resolvers.html#Maven) instead of a [URL-repository resolver](http://www.scala-sbt.org/0.13/docs/Resolvers.html#URL):

```scala
resolvers += "DynamoDBLocal" at "https://s3-us-west-2.amazonaws.com/dynamodb-local/release"
```

**Comment by amcp** [comment]
Also, the latest version of DynamoDB Local available in our repository is 1.11.86.
